widgets/.npmignore

```diff
@@ -0,0 +1,8 @@
+/tests
+
+.storybook/
+/src/stories
+
+
+.env.*
+.env
```

Comment:
```suggestion
.env
.idea
dist
node_modules
```

Reply:
dist needs to not be included. This is what is NOT sent to the NPM repository